source/googleAnalytics.brs

```diff
@@ -182,5 +181,13 @@ Function gamobileSendHit(hit_params As String) As Void
 
   ' Increment the cache buster
   m.gamobile.next_z = m.gamobile.next_z + 1
-  
+
+End Function
+
+Function tostr(input_string)
```

Comment:
I missed that this was a requirement.  It is defined in generalUtils.brs from the Roku SDK.  Since url.brs also needs generalUtils.brs for its itostr() calls, we should just bring in generalUtils.brs to this pull request and get rid of this Function def here.